src/d3a/models/myco_matcher/myco_external_matcher.py

```diff
@@ -74,10 +76,11 @@ def publish_offers_bids(self, message):
                 continue
             for market in area_data["markets"]:
                 # Cache the market (needed while matching)
-                self.markets_mapping[market.id] = market
+                self.area_markets_mapping[f"{area_uuid}-{market.time_slot_str}"] = market
```

Comment:
I know, but I thought that we could at least internally stick with the market_id. But I get now, that it will be complicated when matching the recommendations later.